### ISSUE TITLE
PCF8575 Buttons

### ISSUE THREAD

**Issue by Nexis81** [neu-rah/ArduinoMenu#400]
Is there any way to use buttons connected to a PCF8757 IO expander to navigate the menu?

**Comment by neu-rah** [comment]
yes, issue commands to the `nav` object as suitable for your input, example here:

https://github.com/neu-rah/ArduinoMenu/blob/master/examples/codeCtrl/codeCtrl/codeCtrl.ino
